zulipterminal/ui.py

```diff
@@ -79,6 +79,28 @@ def set_footer_text(self, text_list: Optional[List[Any]]=None,
             time.sleep(duration)
             self.set_footer_text()
 
+    @asynch
+    def set_typeahead_footer(self, suggestions: List[Any], state: int) -> None:
+        try:
+            suggestion = suggestions[state]
+        except (IndexError, TypeError):
+            suggestion = None
```

Comment:
Is this a duplication of, or replacing, the code you just refactored in the other PR?

Reply:
I've dealt with this in the new refactor for `generic_autocomplete`